scene/animation/animation_tree.h

```diff
@@ -164,6 +166,14 @@ class AnimationRootNode : public AnimationNode {
 	AnimationRootNode() {}
 };
 
+class AnimationNodeStart : public AnimationRootNode {
```

Comment:
Maybe they should be StartState and EndState ?

Reply:
Classes renamed. I also renamed the `MultiAnimationNodeStateMachineTransition` to `MultiTransitionEdit`